seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -337,3 +337,17 @@ bnb,onepunchswap,normal,2023-01-12,0xa723fca3a321586a38c738a1bc221a4e9bf4f5e7732
 arbitrum,kyberswap,elastic,2023-04-11,0xdf4e944f86454b662499e8326333dce6af1c18f9663d1289c89b5d49a18d5e9f,5,0x912ce59144191c1204e64559fe8253a0e49e6548,579.3333333,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0.370589731
 arbitrum,kyberswap,elastic,2023-03-11,0xbfd464a0d559cceebbdd20c42ee8a1054de12c936396adbec4bafad9eba60faa,13,0xff970a61a04b1ca14834a43f5de4533ebddb5cc8,2468.999762,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,1.723592528
 arbitrum,kyberswap,elastic,2023-03-26,0x070e704ea6da35b05c97e2a2f9ccf01136f9bdf6d411da31206a837d373e58ca,6,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0.0029,0xff970a61a04b1ca14834a43f5de4533ebddb5cc8,5.145281
+bnb,sushiswap,1,2021-11-01 0:00,0xc66fb80242b7c788c2896b3dc8c611203771be9474d18f85cb2032d594de327c,830,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.07525109648,0x049d68029688eabf473097a2fc38ef61633a3c7a,43.351292
+bnb,sushiswap,1,2021-11-01 0:00,0xffc3e47949b600ee61d7b3b7b57da6137ed4c55652da396d02ff3ece60bfdd9b,591,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.01789196956,0x049d68029688eabf473097a2fc38ef61633a3c7a,9.541071
+polygon,sushiswap,1,2021-06-16 0:00,0xc625dc2c5e923afd8469e9e03233d37eab64d627a4e6f221c8b63efcf276523c,179,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.1131447356,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,191.734491
+polygon,sushiswap,1,2021-06-16 0:00,0xa727f3165a6e7533cbeecfbe67e4072f8b9a0eec486a4c9976b25b902a1bd6f0,310,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.0008197673253,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,1.388941988
+
+bnb,sushiswap,1,2022-11-22,0x683b4299466b4fec3fac2859e2dd27215f69879b31a7384593b46aef71952ed5,373,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,6571.818499746365907562,0xe9e7cea3dedca5984780bafc599bd69add087d56,20
+bnb,sushiswap,1,2022-11-22,0xa572f3ea4bcc491c3d6a3b54f2cd59c776238c3187c1ee7975d9635f8a55d129,260,0xe9e7cea3dedca5984780bafc599bd69add087d56,30.330984668706979348,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,10000
+bnb,sushiswap,1,2022-11-22,0xfac7881f1461c9c56a11f68647385ea27c4d66849f646d5f377e558014652df2,536,0xe9e7cea3dedca5984780bafc599bd69add087d56,0.003061259047161221,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,1
+bnb,sushiswap,1,2023-01-17,0x5169181339080e2d56c13c0a075dac19f19e3f8300a8c068e8f4519bf424e7ab,186,0xe9e7cea3dedca5984780bafc599bd69add087d56,2.487085655648097925,0xc598275452fa319d75ee5f176fd3b8384925b425,177.678
```

Comment:
Edge case 1:
https://bscscan.com/tx/0x5169181339080e2d56c13c0a075dac19f19e3f8300a8c068e8f4519bf424e7ab
This seed is currently not covered by the model.

Reply:
Sorry, the `ext_index` for this seed record should be `166` instead of `186`